src/layouts/AllServices.tsx

```diff
@@ -0,0 +1,542 @@
+import React from 'react';
+
+import {
+  Card,
+  CardBody,
+  CardTitle,
+  Gallery,
+  Masthead,
+  Page,
+  PageSection,
+  PageSectionVariants,
+  Stack,
+  StackItem,
+  Text,
+  TextContent,
+  TextVariants,
+  Title,
+} from '@patternfly/react-core';
+
+import { Header } from '../components/Header/Header';
+import RedirectBanner from '../components/Stratosphere/RedirectBanner';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCartShopping, faChartLine, faCloudArrowUp, faCog, faCreditCard } from '@fortawesome/free-solid-svg-icons';
+import { faSpaceAwesome } from '@fortawesome/free-brands-svg-icons';
```

Comment:
These can be completely replaced by PF's icons
```suggestion
import CreditCardIcon from '@patternfly/react-icons/dist/js/icons/credit-card-icon';
import CogIcon from '@patternfly/react-icons/dist/js/icons/cog-icon';
import CloudUploadAltIcon from '@patternfly/react-icons/dist/js/icons/cloud-upload-alt-icon';
import ChartLineIcon from '@patternfly/react-icons/dist/js/icons/chart-line-icon';
import CartShoppingIcon from '@patternfly/react-icons/dist/js/icons/cart-shopping-icon';
```